ui/tests/integration/components/json-editor-test.js

```diff
@@ -122,12 +122,12 @@ module('Integration | Component | json-editor', function (hooks) {
     assert.dom('.CodeMirror-code').hasText(`{ "test": "********"}`, 'shows data with obscured values');
     assert.dom('[data-test-toggle-input="revealValues"]').isNotChecked('reveal values toggle is unchecked');
     await click('[data-test-toggle-input="revealValues"]');
-    assert.dom('.CodeMirror-code').hasText(JSON_BLOB, 'shows data with real values');
+    assert.dom('.CodeMirror-code').hasText('{ "test": "test"}', 'shows data with real values');
```

Comment:
Can we update this to still use `JSON_BLOB`? Since we changed the functionality of codemirror and no longer update linting on change (which is I think what we were sort of doing before by making brackets on consistent lines) I think the following makes it easier to understand _how_ the codemirror value is different from what was set
```suggestion
    assert.dom('.CodeMirror-code').hasText(JSON.stringify(JSON.parse(JSON_BLOB)), 'shows data with real values');
```

Reply:
This was a chicken or egg for me. If I parse, I still have the issue of an extra space. See screenshot below where I parsed as suggested. I could change the JSON_BLOB const but figured that value was used by other tests. 
![image](https://github.com/hashicorp/vault/assets/6618863/3d15ddfd-b0fc-4d7a-883b-a780a2f2b451)

Here's what it looks like if I make no changes, 
![image](https://github.com/hashicorp/vault/assets/6618863/42d747b2-ae56-413b-bfb9-4ba8d54b06a3)

The difference is a small space at the end, which I think has more to do with [hasText](https://github.com/mainmatter/qunit-dom/issues/186#issuecomment-438411558) than what's being changed. I think the way around this is to use codemirror.getValue and strict equals. Similar to the other kv workflow tests, but that goes against the pattern in this tests. 🤷‍♀️